nextflow.config

```diff
@@ -40,12 +40,17 @@ params {
 
   // Options: Variant calling
   callers = 'varscan2,ivar,bcftools'
-  ivar_exclude_reads = false
+  ivar_trim_noprimer = false
+  ivar_trim_min_len = 20
+  ivar_trim_min_qual = 20
+  ivar_trim_window_width = 4
   filter_dups = false
   filter_unmapped = false
+  mpileup_depth = 0
   min_base_qual = 20
   min_coverage = 10
-  max_allele_freq = 0.8
+  min_allele_freq = 0.25
+  max_allele_freq = 0.75
```

Comment:
Why did you changed the default value?

Reply:
Ah, it is just the same value as is used by the [COG-UK pipeline](https://github.com/connor-lab/ncov2019-artic-nf/blob/d0fd43c1657483af3711812adff51cca678ec550/conf/illumina.config#L40). Shouldnt make a massive difference because anything real should be close to homozygous anyway 🙂